__init__.py

```diff
@@ -72,15 +72,33 @@ def _fbrowser_success(self, instance):
 from kivy.lang import Builder
 from kivy.utils import platform
 from kivy.clock import Clock
+from kivy.compat import PY2
 import string
-from os.path import sep, dirname, expanduser, isdir
+from os.path import sep, dirname, expanduser, isdir, join
 from os import walk
+from sys import getfilesystemencoding
 from functools import partial
 
 if platform == 'win':
     from ctypes import windll, create_unicode_buffer
 
 
+def get_home_directory():
+    if platform == 'win':
+        user_path = expanduser('~')
+
+        if not isdir(join(user_path, 'Desktop')):
+            user_path = dirname(user_path)
+
+    else:
+        user_path = expanduser('~')
+
+    if PY2:
```

Comment:
These days I think that `if isinstance(..., bytes)` is preferred to `if PY2` because it's more specific, but I guess it's fine.